readme.md

```diff
@@ -81,10 +81,48 @@ See: https://github.com/praegus/toolchain-fitnesse-plugin
 ![alt text](https://github.com/tcnh/binstore/raw/master/macen_versioncheck.png "Maven dependency versions")
 
 #### ToolTips
-* Tooltips now display on every page
+Tooltips are now displayed on every page.
+
+You can add your own tooltips with the following steps:
+1. You go to fitnesse/resources/bootstrap-plus/txt/toolTipData.txt for editing.
+2. Put the text at the end of the file and separate the tooltips by enters
+3. The format of a tooltip is[this is a tooltip] (without the brackets).
+
+Tooltips can now contain links
+* You can add a link to your tooltip by adding a simple html link tag to your tooltip.
+
+Tooltips will now also load in from fixtures:
+
+* You can add tooltips to your fixture by adding a Tooltips.txt on [FixtureRoot]/[FixtureName]_tooltips/Tooltips.txt
+* You also need to add two executions to your maven dependency plugin to be able to load fixture tooltips:
+
+                    <execution>
+                        <id>unpack-dependencies</id>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>unpack-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <includes>**/Tooltips.txt</includes>
+                            <outputDirectory>${project.basedir}/wiki/TooltipData</outputDirectory>
```

Comment:
Zet deze de tooltips in een directory per dependency?

Reply:
Hij haalt de tooltips.txt en alle mappen waar ie in staat uit de fixture, we hebben voor die reden ook in onze instructies staan om de tooltip.txt niet in de root van de fixture te plaatsen maar een mapje te maken en daarin de tooltips.txt te plaatsen